core/src/main/scala/akka/persistence/cassandra/query/EventsByPersistenceIdStage.scala

```diff
@@ -320,14 +347,26 @@ import com.datastax.driver.core.utils.Bytes
                 // skip event due to fast forward
                 tryPushOne()
               } else if (config.gapFreeSequenceNumbers && seqNr != event.sequenceNr) {
-                failStage(new IllegalStateException(s"Missing sequence number [$seqNr], got [${event.sequenceNr}] for " +
-                  s"persistenceId [$persistenceId]."))
+                lookingForMissingSeqNr match {
+                  case Some(_) => throw new IllegalStateException(
+                    s"Should not be able to get here when already looking for missing seqNr [$seqNr] for entity [$persistenceId]"
+                  )
+                  case None =>
+                    lookingForMissingSeqNr = Some(
+                      MissingSeqNr(Deadline.now + config.eventsByPersistenceIdEventTimeout, event.sequenceNr)
+                    )
+                    query(false)
```

Comment:
should it query immediately, or `scheduleOnce(LookForMissingSeqNr, 200.millis)` ?

Reply:
Delaying it is probably a good idea as we just saw it missing.